tests/2_docker_daemon_configuration.sh

```diff
@@ -5,7 +5,7 @@ info "2 - Docker Daemon Configuration"
 
 # 2.1
 check_2_1="2.1  - Do not use lxc execution driver"
-pgrep -lf docker | grep lxc >/dev/null 2>&1
+get_command_line_args docker | grep lxc >/dev/null 2>&1
```

Comment:
As these updated checks will look for the pattern (i.e. `lxc`) on a single line, can the pattern be improved to match the entire line, or as much of the line is reasonable? i.e. `^lxc`, `lxc$`, etc. Similar changes for each of the patterns below as well.